StoreSpidLogs/index.ts

```diff
@@ -1,11 +1,23 @@
 import { Context } from "@azure/functions";
+import { getRequiredStringEnv } from "io-functions-commons/dist/src/utils/env";
 import { AzureContextTransport } from "io-functions-commons/dist/src/utils/logging";
 import * as t from "io-ts";
 import { UTCISODateFromString } from "italia-ts-commons/lib/dates";
 import { readableReport } from "italia-ts-commons/lib/reporters";
 import { IPString, PatternString } from "italia-ts-commons/lib/strings";
+import * as NodeRSA from "node-rsa";
 import * as winston from "winston";
 
+const rsaPublicKey = getRequiredStringEnv("RSA_PUBLIC_KEY");

```

Comment:
SPID_LOGS
```suggestion
const rsaPublicKey = getRequiredStringEnv("SPID_LOGS_PUBLIC_KEY");

```

Reply:
Is it OK to declare encryptionKey directly? e.g: `const rsaPublicKey = new NodeRSA(getRequiredStringEnv("SPID_LOGS_PUBLIC_KEY"));`

Reply:
it depends from the implementation of node-rsa, if `rsaPublicKey` keeps an internal state (I don't know) you must create a new key for each encryption